.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,17 +8,19 @@ assignees: ''
 ---
 
 <!--
-Before opening an issue, please take a look at the [troubleshooting guide](https://haskell-language-server.readthedocs.io/en/latest/troubleshooting.html). This explains some common issues and will also help you to find the information that the issue template asks for.
+Before opening an issue, please take a look at the [troubleshooting guide](https://haskell-language-server.readthedocs.io/en/latest/troubleshooting.html).
+This explains some common issues and will also help you to find the information that the issue template asks for.
 
-When filing an issue, please fill out as much of the information below as you can. This helps us to debug your issue, but is not required!
+When filing an issue, please fill out as much of the information below as you can.
+This helps us to debug your issue, but is not required!
 -->
 
 ### Your environment
 
 Which OS do you use:
 <!-- Windows, MacOS, Ubuntu, ArchLinux, etc... -->
-Which lsp-client do you use:
-<!-- Neovim, emacs, VS Codium, etc... -->
+Which LSP client (editor/plugin) do you use:
+<!-- Neovim+LanguageClient-neovim, emacs+lsp-mode, VS Codium+vscode-haskell, etc... -->
```

Comment:
If I may: IIRC LanguageClient-neovim is deprecated long time ago and was incorporated into neovim itself. So majority modern neovim setups use either builtin lsp or coc.nvim.

Reply:
Thanks!